src/main/java/pl/edu/mimuw/queue/IntQueueNode.java

```diff
@@ -16,6 +16,35 @@ public IntQueueNode(int value) {
     this(value, null, null);
   }
 
-  // TODO: make changes with this class fields, constructors, methods -
-  //  this is just a helper class for your implementation that can be modified as you wish
+  public int value() {return this.value;}
+
+  public IntQueueNode next() {return this.next;}
+
+  public IntQueueNode prev() {return this.prev;}
+
+  /**
+   * Searches through {@code next} references until it encounters a node with
+   * empty {@code next} reference (that is, the end of the queue). It then
+   * extends that queue by adding a new end node.
+   */
+  public void attachToEnd(IntQueueNode newEnd) {
```

Comment:
mam nadzieję że masz świadomość nieefektywności tego, choć nie było to wymagane 😉

Reply:
Poprawione... Mialem najwyraźniej potężne zaćmienie umysłu